swap_meet/vendor.py

```diff
@@ -1,2 +1,82 @@
+from swap_meet.item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self,inventory=None):
+        self.inventory= inventory
+        self.inventory= [] if self.inventory is None else self.inventory
+        
+    def add(self,item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self,item):
+        if item not in self.inventory:
+            return False
+        else:
+            self.inventory.remove(item)
+        return item
```

Comment:
We could remove the `else` statement without changing the outcome. This would be preferred so the main flow of the code is as un-indented and prominent as possible.

```py
        if item not in self.inventory:
            return False

        self.inventory.remove(item)
        return item
```